homework/index.js

```diff
@@ -0,0 +1,74 @@
+let repositories;
+async function getPromise() {
+  const res = await fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100');
+  return res.json();
+}
+
+const myPromise = getPromise(); // Set the promise as a global variable
+
+async function selectedRepoCaracteristics(selectValue) {
```

Comment:
Your function name should reflect what it *does*. In this case, maybe `showSelectedRepoDetails`? (Also, avoid English language mistakes (it's 'characteristics') at all cost in code. If unsure, double-check the word's spelling on Google; it's super quick!)